client/fingerprint/network_unix.go

```diff
@@ -30,21 +30,41 @@ func NewNetworkFingerprinter(logger *log.Logger) Fingerprint {
 	return f
 }
 
+func flagsSet(flags net.Flags, test net.Flags) bool {
+	return flags&test != 0
+}
+
+func flagsClear(flags net.Flags, test net.Flags) bool {
```

Comment:
> This makes Nomad not connect to loopback device by default. People might use loopback while testing but not in prod. Not sure if we should make this a default behavior.

I didn't use loopback as default, since current implementation is so. - https://github.com/hashicorp/nomad/blob/1a97004398b2306634b2c9005f9e77f38faaa44e/client/fingerprint/network_unix.go#L37-L41
Should I change it?